samples/bot-message-reminder/csharp/README.md

```diff
@@ -0,0 +1,124 @@
+  ```bash
+  ngrok http -host-header=localhost 3978
+  ```
+- [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+
+## To try this sample
+
+1) Setup for Bot
+   In Azure portal, create a [Azure Bot resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2).
+   
+2) Clone the repository
+   ```bash
+   git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+   ```
+   
+3) In folder where repository is cloned, navigate to `samples/bot-message-reminder/csharp`
+
+    change into project folder
+    ```bash
+    cd # MessagingExtensionReminder
+    ```
+    
+4) Run ngrok - point to port 3978
+
+    ```bash
+    # ngrok http -host-header=rewrite 3978
+    ```
+ 
+5) Modify the `manifest.json` in the `/AppPackage` folder and replace the following details:
+  - `{{Microsoft-App-Id}}` with Application id generated from Step 1
+  - `{{domain-name}}` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok.io` then your domain-name will be *.ngrok.io.
+
+6) Zip the contents of `AppPackage` folder into a `manifest.zip`, and use the `manifest.zip` to deploy in app store or add to Teams using step 9.
+
+7) Modify the `/appsettings.json` and fill in the following details:
+  - `{{Microsoft-App-Id}}` - Generated from Step 1 is the application app id
+  - `{{ Microsoft-App-Password}}` - Generated from Step 1, also referred to as Client secret
+  - `{{ Application Base Url }}` - Your application's base url. E.g. https://12345.ngrok.io if you are using ngrok.
+
+  
+8) Run the bot from a terminal or from Visual Studio, choose option A or B.
+ 
+   A) From a terminal
+     ```bash
+     # run the bot
+     dotnet run
+     ```
+
+   B) Or from Visual Studio
+     - Launch Visual Studio
+     - File -> Open -> Project/Solution
+     - Navigate to `MessagingExtensionReminder` folder
+     - Select `MessagingExtensionReminder.csproj` file
+     - Press `F5` to run the project 
+
+9) Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
+   - Go to Microsoft Teams. From the lower left corner, select Apps
+   - From the lower left corner, choose Upload a custom App
+   - Go to your project directory, the ./AppPackage folder, select the zip folder, and choose Open.
+   - Select Add in the pop-up dialog box. Your app is uploaded to Teams.
+
+## Features of this sample
+
+Personal scope scenario 
+
+![Select message](MessagingExtensionReminder/Images/MessageAction.png)
+
+![Task Details](MessagingExtensionReminder/Images/ScheduleTask.png)
+
+![Task reminder](MessagingExtensionReminder/Images/TaskReminder.png)
+
+Team scope scenario
+
+![Team message action ](MessagingExtensionReminder/Images/MessageActionTeam.png)
+
+![Team Task Details ](MessagingExtensionReminder/Images/ScheduleTaskTeam.png)
+
+![ TeamTask reminder](MessagingExtensionReminder/Images/TaskReminderTeam.png)
```

Comment:
Add proper headers or explanation for every image properly.

Reply:
added headers